spec/services/idv/session_spec.rb

```diff
@@ -119,10 +135,16 @@
 
         it 'sets profile to pending in person verification' do
           subject.create_profile_from_applicant_with_password(user.password)
+          profile = subject.profile
 
           expect(subject).not_to have_received(:move_pii_to_user_session)
-          expect(subject.profile.active?).to eq(false)
-          expect(subject.profile.deactivation_reason).to eq('in_person_verification_pending')
+          expect(profile.activated_at).to eq nil
+          expect(profile.active).to eq false
+          expect(profile.deactivation_reason).to eq 'in_person_verification_pending'
```

Comment:
Is there a reason this is a string instead of a keyword? I'm surprised this passes!
```suggestion
          expect(profile.deactivation_reason).to eq :in_person_verification_pending
```

Reply:
Your change will fail, actually! Adding though, for completeness.